### PR TITLE
Add map components

### PR DESCRIPTION
**Pull request by augusto-jm-amaral**

### CODE REVIEW

File: src/components/TechnicalCard/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    companyName: '',
+    counter: '',
+  },
+  status: {
+    type: '',
+    start: '',
+  },
+}
+
+TechnicalCard.propTypes = {
```
Comment (alexandrejuk) [inline src/components/TechnicalCard/index.js]: I think that should have the technician id.

**Review by alexandrejuk — CHANGES_REQUESTED** [review]

File: src/components/TechnicalCard/index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Card, Avatar } from 'antd'
+import styles from './index.css'
+
+const cardBodyStyle = {
```
Comment (alexandrejuk) [inline src/components/TechnicalCard/index.js]: this style css should be here?

File: src/components/TechnicalCard/index.js (hunk 1 above)
Comment (augusto-jm-amaral) [reply src/components/TechnicalCard/index.js]: Yah, I will change it, thanks.

**Review by alexandrejuk — APPROVED** [review]

File: src/components/TechnicalCard/index.js (hunk 2 above)
Comment (augusto-jm-amaral) [reply src/components/TechnicalCard/index.js]: Card from ant designer receive only object as bodyStyle 😢

**Pull request merged**